ansible/roles/analytics-spark-provision/templates/spark-env.j2

```diff
@@ -74,3 +74,5 @@ export reports_storage_key={{sunbird_private_storage_account_name}}
 export reports_storage_secret={{sunbird_private_storage_account_key}}
 export azure_storage_key={{sunbird_private_storage_account_name}}
 export azure_storage_secret={{sunbird_private_storage_account_key}}
+export public_reports_storage_key={{sunbird_public_storage_account_name}}
```

Comment:
can we rename it to druid_storage_account_key instead of public_?

Reply:
Ya, we can do that.